scripts/create_provider_artifact.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+import sys
+from pathlib import Path
+
+
+ALL_PROVIDERS = ["loki", "elasticsearch", "cloudwatch", "https", "sentry", "prometheus"]
```

Comment:
how about auto discovering all providers in that dir? then we wouldn't have to change this when publishing a new provider

ofc we have to add exception for the example provider

Reply:
I could invert yeah, but I feel like it's easier to remember to add a new provider here, rather than exclude a directory manually (currently `sample` and `grafana-common` need to be excluded).

I just discovered that `cargo metadata` is an actual workhorse, so I'll try to rely on that to produce the list